src/scss/templates/_blog.scss

```diff
@@ -160,3 +160,46 @@
 	}
 
 }
+
+.blog-post-quick-links {
+	color: $body-color;
+	border-bottom: $nav-tabs-border-width solid $nav-tabs-border-color;
+	&:hover {
+		text-decoration: none;
+	}
+	& + .blog-post-quick-links {
+		&::before {
+			content: '|';
+			float: none;
+		}
+	}
+}
+
+.blog-back-to-top {
+	position: fixed;
+	bottom: calc-em(20);
+    right: calc-em(15);
+    cursor: pointer;
+    z-index: $zindex-dropdown;
+	opacity: 0;
+	transition: 0.4s ease-in;
+	transition-property: opacity, bottom;
+	display: none;
+
+	&:hover {
+		color: $primary;
+	}
+
+	&.btt-btn--show {
+		opacity: 1;
+		display: flex;
+	}
+
+	.svg {
+		font-size: calc-em(40);
+	}
+
+	@include media-breakpoint-up(lg) {
+		right: calc-em(40);
+	}
```

Comment:
this should go after the main class properties not at its end

Reply:
> this should go after the main class properties not at its end

Breakpoint has been moved